docs/time_series_performance_benchmarks.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>Suggested rewording: "The response time increases with the size of the area of interest, especially for higher resolution images"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>